extension/docs/HowToRun.md

```diff
@@ -43,113 +43,60 @@ Extension module requires 1 environment variable to start. This environment vari
+| `commercetools` | `apiUrl`                     | The commercetools HTTP API is hosted at that URL.                                                                                                                                                                                                                                                                                                                                                                        | `https://api.europe-west1.gcp.commercetools.com`                                                                                                                                        |
+| `commercetools` | `authUrl`                    | The commercetools’ OAuth 2.0 service is hosted at that URL.                                                                                                                                                                                                                                                                                                                                                              | `https://auth.europe-west1.gcp.commercetools.com`                                                                                                                                       |
+| `commercetools` | `authentication`             | This setting only takes effect when `basicAuth` ( a child attribute in `ADYEN_INTEGRATION_CONFIG` ) is set to `true`. It enables authentication mechanism to prevent unauthorized access to the extension module. When it is provided as a JSON object, it must contain 3 separate attributes. They are `scheme` attribute which supports `basic` type, `username` and `password` attribute defined by user.             |                                                                                                                                                                                         |
+| `other`         | `basicAuth`                  | Boolean attribute to enable/disable basic authentication to prevent unauthorized 3rd-party from accessing extension endpoint                                                                                                                                                                                                                                                                                             | false                                                                                                                                                                                   |
```

Comment:
Attribute group and `other` needs to be explained more visible and explicit what they mean. I saw there is a small mention about it above `Other configurations can be set as ...`, but I don't think this is enough for people to understand.

Maybe make a small section with name `Attribute group` just above these tables and write there what every group means?

Reply:
How does it look like now: https://github.com/commercetools/commercetools-adyen-integration/pull/938/commits/92c776a76280f35c0658c2dd97eaa69f06fd9601